node/src/components/rpc_server/rpcs/state.rs

```diff
@@ -929,6 +927,141 @@ impl RpcWithParamsExt for QueryGlobalState {
     }
 }
 
+/// Identifier for the main purse of a given account.
+#[derive(Serialize, Deserialize, Debug, Clone, JsonSchema)]
+#[serde(deny_unknown_fields, rename_all = "snake_case")]
+pub enum BalanceIdentifier {
```

Comment:
I think a better name is `PurseIdentifier`.  Please ensure all variables' names are updated as well as the changelog, doc comments, etc.

Reply:
Renamed to PurseIdentifier, alongside updates in the changelog